src/cljs/athens/views/blocks/core.cljs

```diff
@@ -255,13 +271,12 @@
                      (or (and (true? linked-ref) (:linked-ref/open @state))
                          (and (false? linked-ref) open)))
             (for [child children]
-              [:div {:key (:db/id child)}
+              [:<> {:key (:db/id child)}
```

Comment:
Totall unnecessary div here

Reply:
If div is unnecessary, is fragment also unnecessary?

Reply:
The fragment includes the key, which is necessary.

Reply:
Got it. Suppose we could add a key somehow without a fragment -- Do you think there would be a performance boost removing the fragment?

Reply:
I think the fragment is the way we add the key, and the fragment exists only in the Reagent tree, not the DOM, so shouldn't result in any performance hit.